src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -0,0 +1,294 @@
+﻿<Window xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:Class="Dynamo.Wpf.Views.PreferencesWindow"
+        x:Name="DynamoPreferencesWindow"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:p="clr-namespace:Dynamo.Wpf.Properties;assembly=DynamoCoreWpf"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:views="clr-namespace:Dynamo.Wpf.Views"
+        WindowStartupLocation="CenterOwner"
+        WindowStyle="None"
+        ResizeMode="NoResize"
+        mc:Ignorable="d" 
+        Height="474" 
+        Width="530" >
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <SolidColorBrush x:Key="PreferencesWindowBackgroundColor" 
+                             Color="#535353" />
+            <SolidColorBrush x:Key="PreferencesWindowFontColor" 
+                             Color="#E5E5E5" />
+            <SolidColorBrush x:Key="PreferencesWindowButtonColor" 
+                             Color="#0696D7" />
+            <SolidColorBrush x:Key="PreferencesWindowButtonPressedColor" 
+                             Color="#0778B0" />
+            <SolidColorBrush x:Key="PreferencesWindowButtonMouseOverFontColor" 
+                             Color="#CDCBCB" />
+            <SolidColorBrush x:Key="CloseButtonMouseOverBackground" 
+                             Color="#E81123" />
+            <SolidColorBrush x:Key="CloseButtonPressed" 
+                             Color="#B70F1B" />
+            <SolidColorBrush x:Key="CloseButtonMouseOverForeground" 
+                             Color="#FFFFFF" />
+            <SolidColorBrush x:Key="TabItemUnSelected" 
+                             Color="DarkGray" />
```

Comment:
Thanks! Check for Dynamo/src/DynamoCoreWpf/UI/Themes/Modern, I think most Dynamo styles are stored there, if possible I would prefer styles not stored in the modal dialog xaml. Would you look at it?

Reply:
@QilongTang I moved all the Styles to the src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml file. Thanks